www/include/monitoring/downtime/xml/broker/makeXMLForDowntime.php

```diff
@@ -120,21 +120,23 @@
 }
 
 $rowClass = "list_one";
-while ($row = $stmt->fetch()) {
-    $row['comment_data'] = strip_tags($row['comment_data']);
-    $xml->startElement('dwt');
-    $xml->writeAttribute('class', $rowClass);
-    $xml->writeElement('author', $row['author']);
-    $xml->writeElement('start', $row['actual_start_time']);
-    if (!$row['fixed']) {
-        $row['end_time'] = (int)$row['actual_start_time'] + (int)$row['duration'];
+if (isset($res)) {
```

Comment:
```suggestion
if ($res === true) {
```

Reply:
I forgot to remove debug from last investigations

Reply:
-> https://github.com/centreon/centreon/pull/8708#discussion_r423620221